cogboard-webapp/src/components/widgets/dialogFields/ToDoListinput.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  IconButton,
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText
+} from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import CheckIcon from '@material-ui/icons/Check';
+import { remove } from 'ramda';
+import { prepareChangeEvent } from './helpers';
+import AddIcon from '@material-ui/icons/Add';
+import Input from '@material-ui/core/Input';
+import Fab from '@material-ui/core/Fab';
+import styled from '@emotion/styled/macro';
+
+const StyledInput = styled(Input)`
+  margin-top: 16px;
+  margin-bottom: 8px;
+`;
+
+const StyledFab = styled(Fab)`
+  margin-top: 16px;
+  margin-bottom: 8px;
+`;
+
+const StyledList = styled(List)`
+  margin-top: 16px;
+`;
+
+const ToDoListInput = ({ value, onChange }) => {
+  const [formValueItemText, setFormValueItemText] = useState('');
+  const [editMode, setEditMode] = useState(false);
+  const handleChangeValItemText = event =>
+    setFormValueItemText(event.target.value);
+  const [items, setItems] = useState(() =>
+    (value || []).map(item => {
+      return {
+        id: item.id,
+        itemText: item.itemText
+      };
+    })
+  );
+
+  console.log(items);

```

Comment:
👎

Reply:
Yup, this is for test purpose. This will not be the final code